src/audio/volume/volume.c

```diff
@@ -385,13 +409,34 @@ static struct comp_dev *volume_new(const struct comp_driver *drv,
 		cd->muted[i] = false;
 	}
 
+	switch (vol->ramp) {
+	case SOF_VOLUME_LINEAR:
+	case SOF_VOLUME_LINEAR_ZC:
+		cd->ramp_func = &volume_linear_ramp;
+		break;
+	case SOF_VOLUME_WINDOWS_FADE:
+		cd->ramp_func = &volume_windows_fade_ramp;
+		break;
+	case SOF_VOLUME_LOG:
```

Comment:
why define these if they are not implemented?

Reply:
Yep. I removed the log ramp references.